hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieCompactedLogRecordScanner.java

```diff
@@ -72,142 +78,148 @@
-                  HoodieTimeline.LESSER_OR_EQUAL)) {
-            //hit a block with instant time greater than should be processed, stop processing further
-            break;
-          }
-          switch (r.getBlockType()) {
-            case AVRO_DATA_BLOCK:
-              log.info("Reading a data block from file " + logFile.getPath());
-              // If this is a avro data block, then merge the last block records into the main result
-              merge(records, lastBlocks);
-              // store the last block
-              lastBlocks.push(r);
-              break;
-            case DELETE_BLOCK:
-              log.info("Reading a delete block from file " + logFile.getPath());
-              String lastBlockInstantTime = lastBlocks.peek().getLogMetadata().get(INSTANT_TIME);
-              if(!lastBlockInstantTime.equals(blockInstantTime)) {
-                // Block with the keys listed as to be deleted, data and delete blocks written in different batches
-                // so it is safe to merge
-                // This is a delete block, so lets merge any records from previous data block
-                merge(records, lastBlocks);
-              }
-              // store deletes so can be rolled back
-              lastBlocks.push(r);
-              break;
-            case COMMAND_BLOCK:
-              log.info("Reading a command block from file " + logFile.getPath());
-              // This is a command block - take appropriate action based on the command
-              HoodieCommandBlock commandBlock = (HoodieCommandBlock) r;
-              String targetInstantForCommandBlock = r.getLogMetadata().get(HoodieLogBlock.LogMetadataType.TARGET_INSTANT_TIME);
-              switch (commandBlock.getType()) { // there can be different types of command blocks
-                case ROLLBACK_PREVIOUS_BLOCK:
-                  // Rollback the last read log block
-                  // Get commit time from last record block, compare with targetCommitTime, rollback only if equal,
-                  // this is required in scenarios of invalid/extra rollback blocks written due to failures during
-                  // the rollback operation itself
-                  HoodieLogBlock lastBlock = lastBlocks.peek();
-                  if (lastBlock != null && lastBlock.getBlockType() != CORRUPT_BLOCK &&
-                        targetInstantForCommandBlock.contentEquals(lastBlock.getLogMetadata().get(INSTANT_TIME))) {
-                    log.info("Rolling back the last log block read in " + logFile.getPath());
-                    lastBlocks.pop();
-                  } else if(lastBlock != null && lastBlock.getBlockType() == CORRUPT_BLOCK) {
-                    // handle corrupt blocks separately since they may not have metadata
-                    log.info("Rolling back the last corrupted log block read in " + logFile.getPath());
-                    lastBlocks.pop();
+      while(logFilePathsItr.hasNext()) {
```

Comment:
whats the actual change in these lines, seems same to me? seems like whitespace or indentation?

Reply:
Result of merge conflicts.